src/app/core/component/graph-display/graph-display.component.ts

```diff
@@ -196,13 +218,33 @@ export class GraphDisplayComponent implements AfterViewInit, OnChanges {
         this.startSimulation();
     }
 
-
     backgroundClicked() {
```

Comment:
General question: Did you check if this also fires for clicks in your sidenavs?

Reply:
Yes, it does not fire. It only fires for clicks in the graph, not in the sidenavs.